pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -43,6 +40,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/controller-manager/pkg/informerfactory"
+	"k8s.io/klog/v2"
```

Comment:
Why do we need to modify here?

Reply:
Just a layout style, i think it's better.